client/cli/src/params.rs

```diff
@@ -438,16 +438,30 @@ pub struct RunCmd {
 
 	/// Listen to all RPC interfaces.
 	///
-	/// Default is local.
+	/// Default is local. Note: it isn't safe to expose RPC publicly without a proxy server.
+	/// Use `--unsafe-rpc-external` to suppress the warning if you understand the risks.
 	#[structopt(long = "rpc-external")]
 	pub rpc_external: bool,
 
+	/// Listen to all RPC interfaces.
+	///
+	/// Same as `--rpc-external`.
+	#[structopt(long = "unsafe-rpc-external")]
+	pub unsafe_rpc_external: bool,
+
 	/// Listen to all Websocket interfaces.
 	///
-	/// Default is local.
+	/// Default is local. Note: it isn't safe to expose RPC publicly without a proxy server.
```

Comment:
```suggestion
	/// Default is local. Note: not all RPC methods are safe to be exposed publicly. Use a RPC proxy server to filter out dangerous methods. More details: (link to wiki).
```

Does anyone know if we have node configuration documentation somewhere? CC @shawntabrizi ?

Reply:
Unfortunately, it looks like we don't have such documentation.